src/index.js

```diff
@@ -290,21 +322,27 @@ function clear(element) {
 
```

Comment:
Functionality changes below. (As there are a lot of formatting changes, this is just a comment to anchor to)